2.4/run-mongod.sh

```diff
@@ -7,6 +7,8 @@ set -e
 MAX_ATTEMPTS=60
 SLEEP_TIME=1
 
+MONGODB_CONFIG_PATH="/opt/openshift/etc/mongodb.conf"
```

Comment:
`WORKDIR /opt/openshift`

this means that you can just do:

MONGODB_CONFIG_PATH="./etc/mongodb.conf"